src/locales/cy/pages.yml

```diff
@@ -0,0 +1,130 @@
+landingPage:
+  title: "Sut i brofi eich hunaniaeth mewn Swyddfa Bost"
```

Comment:
We had English on the landing page in CIC just trilpe checking we want welsh here?

Reply:
The text in CIC isn't being used I'll be removing it part of another ticket